src/graphql/resolvers/ssb.js

```diff
@@ -17,6 +26,78 @@ module.exports = function Resolvers () {
     return profile
   }
 
+  /**
+   * Gets all the profile ids of followers of the given profile id
+   * @param {string} id - id of a profile
+   */
+  const getFollowersIds = (id) => {
+    return new Promise((resolve, reject) => {
+      pull(
+        ssb.db.query(
+          where(
+            contact(id)
+          ),
+          descending(), // latest => oldest
+          toPullStream()
+        ),
+        pull.filter(msg => msg.value.content.following),
```

Comment:
:fire: this won't take into account more complex cases where a person follows someone, then unfollows them or blocks them.

Use existing well testing module [`ssb-friends`](https://www.npmjs.com/package/ssb-friends) for this:

```js
ssb.friends.hops({ start: id, max: 1, reverse: true }, (err, res) => {
  resolve(Object.keys(res))  // I think
})
```